43 - Understanding Few-Shot Multi-Task Representation Learning Theory

qbouniot opened the pull request:
## OpenReview Submission Thread

- https://openreview.net/forum?id=JSiFAx0lKMU

## Checklist before requesting a review
- [x] I have followed the instructions and have only added changes as specified in the submission instructions (blog post and asset directories with correct naming scheme)
- [x] I have implemented the necessary changes in response to reviewer feedback
- [x] (optional, for tracking) I deployed github pages on my fork in order to preview my post
    - [x] If yes, I have deleted my hosted github page once this pull request was closed (to be completed upon closing this PR)

## Changes implemented in response to reviewer feedback

- Detailed all author's name in references
- Added a note to say that all assumptions are not required simultaneously in the assumptions' introduction section
- Clarification about necessity of assumption 2.2 (in the blogpost)

## Any other comments

- n/a

busycalibrating reviewed (changes requested):
Looks great! Other than updating the authors list, the only thing I wanted to check was the image references - if there are images you took from other places could you put a source link underneath them? If not then disregard this! 

**Review checklist:**

- [x] Correct naming of post and asset directories
- [x] Assets are in the correct location
- [x] No unnecessary files or modifications to other code
- [x] Post excerpt in the `blog/` endpoint renders well
- [x] No obvious rendering/formatting errors (not a thorough check!)
- [ ] All figures have citations where necessary

qbouniot commented:
Thank you for pointing this out ! I added a source link for the single image taken, since the other ones are original images. I also credited the painters :)
I updated the authors in the header of the blogpost.

busycalibrating commented:
@qbouniot Thanks for the changes! Re: authors, could you change it to follow the format:

`last_name_1, first_name_1 (affiliation_1); last_name_2, first_name_2 (affiliation_2)`

 affiliation is optional, so it could just be:

`Bouniot, Quentin; Redko, Ievgen`

this is necessary for proper bibtex entry generation!

qbouniot commented:
I corrected the format for the authors list, sorry for that !

busycalibrating commented:
No problem, thanks!
